src/main/archetype/ui.permissions/src/main/content/jcr_root/home/groups/__appsFolderName__/custom-content-author/.content.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:rep="internal"
+    jcr:mixinTypes="[rep:AccessControllable]"
+    jcr:primaryType="rep:Group"
+    jcr:uuid="9e6b62c2-6e39-3afc-b711-01e846cd2e35"
+    rep:authorizableId="custom-content-author"
+    rep:principalName="custom-content-author">
+    <profile
+        jcr:mixinTypes="[rep:AccessControllable]"
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="cq/security/components/profile"
+        givenName="${appsFolderName} Content Author"/>
```

Comment:
@bpauli Is there any way to capitalize the first letter of the value for {appsFolderName}

Reply:
If we cant leverage StringUtils capitalize(), then is it ok to use the following:
${appsFolderName.substring(0,1).toUpperCase()}${appsFolderName.substring(1)}